Documentation/getting-started-test-app.md

```diff
@@ -0,0 +1,30 @@
+# Getting started with the test app
+
+## Data Collection mobile property prerequisites
+
+The test app needs to be configured with a Data Collection mobile property with the following extensions before it can be used:
```

Comment:
Can you add a line here, or a note, stating that Edge Bridge does not contain a card/extension in Data Collection? You can use the same note in `getting-started.md` line 21.